components-api/src/main/scala/pl/touk/nussknacker/engine/api/typed/CanBeSubclassDeterminer.scala

```diff
@@ -135,7 +137,11 @@ trait CanBeSubclassDeterminer {
         (),
         f"${givenClass.display} and ${superclassCandidate.display} are not the same"
       ) orElse
-        isAssignable(givenClass.klass, superclassCandidate.klass)
+        condNel(
+          isAssignable(givenClass.klass, superclassCandidate.klass),
```

Comment:
Please use `CanBeSubclassDeterminer.isAssignable` to show that it is our class

Reply:
Ok, changed

Reply:
not appliable aanymore, resolve